src/test/java/teammates/test/cases/browsertests/StudentFeedbackSubmitPageUiTest.java

```diff
@@ -648,6 +723,18 @@ private void testInputValidation() {
 
     }
 
+    private void testLinks() {
```

Comment:
Change not relevant?

Reply:
Yes, although not relevant to this PR, I changed its position according to the order in which these functions are called. :sweat_smile:

Reply:
Please note that tests will no be run in the order of their declaration order in testng. Also, it is a bad practice to introduce the running order (dependency). Do you really need that?

Reply:
Ohh no not really. I thought it would look better to have it in order. Never mind.